install/config/tools/cmd_tlm_server/cmd_tlm_server.txt

```diff
@@ -1,2 +1,4 @@
 TITLE 'COSMOS Command and Telemetry Server'
 AUTO_INTERFACE_TARGETS # Use cmd_tlm_server.txt from all targets declared in system.txt
+
+BACKGROUND_TASK system_background_task.rb
```

Comment:
Accidental commit as I was working on the new System documentation. I'll remove.